apps/wallet/src/types/css.d.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+declare module '*.css' {
```

Comment:
Do you need this here + core?

Reply:
I couldn't find a way to share TypeScript declaration files between projects, but then again I didn't spend a tremendous amount of time on this. But yeah webpack doesn't build correctly without this declaration file